src/Compilers/VisualBasic/Test/Emit/Emit/CompilationEmitTests.vb

```diff
@@ -1060,6 +1060,162 @@ End Class"
         End Sub
 
         <Fact>
+        Public Sub RefAssembly_VerifyTypesAndMembersOnImplementedProperty()
+            Dim source = "
+Public Interface I
+    Property P As Integer
+End Interface
+Public Class C
+    Implements I
+    Property P As Integer Implements I.P
```

Comment:
>Property P As Integer Implements I.P [](start = 4, length = 36)

In VB members are public by default, you might want to mark them as private explicitly for the purpose of this test.